lib/utils.js

```diff
@@ -7,6 +7,12 @@ function isPlainObject(value) {
 }
 exports.isPlainObject = isPlainObject;
 
+// Support dynamic require within webpack bundle.
+if (typeof __non_webpack_require__ === 'undefined') {
+  var __non_webpack_require__ = require;
```

Comment:
> Hmm, I think the presence of this var is going to ensure that __non_webpack_require__ === 'undefined' is always true.

You're right for running code it's always `undefined`. Which is what I want since this is a hint for the webpack bundler when it's running its bundle transform. When the bundle transform happens only the invocations are changed. So `___non_webpack_require__(smth)` is changed to `require(smth)`. If you try to set the variable outside of the block then it loses its webpack magic-ness. I've seen others set it on the `global` property but I wanted to avoid that and after some tinkering found that I could get it to work by using a var in a block.